apps/windows/package.json

```diff
@@ -39,7 +41,9 @@
     "tsconfig-paths": "^3.9.0",
     "typescript": "3.8.3",
     "@types/jasmine": "3.5.10",
-    "webdriverio": "5.22.4"
+    "webdriverio": "5.22.4",
+    "@wdio/allure-reporter": "^5.22.4",
```

Comment:
nit, you could group related deps together (wdio all together) 😃

Reply:
Changing in newest commit